content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1638,6 +1667,7 @@ Defines a [default value for a field](/concepts/components/prisma-schema/data-mo
 
 - Default values that cannot yet be represented in the Prisma schema are represented by the `dbgenerated()` function when you use [introspection](/concepts/components/introspection).
 - Default values are currently not allowed on relation fields in the Prisma schema. Note however that you can still define default values _manually_ in the underlying database using plain SQL. These will not be represented in the Prisma schema though, this is a temporary limitation which you can [track on GitHub](https://github.com/prisma/prisma/issues/1111).
```

Comment:
Unrelated to the scalar list change, but this block should also change since it is out of date. 

`Default values are currently not allowed on relation fields in the Prisma schema. Note however that you can still define default values _manually_ in the underlying database using plain SQL. These will not be represented in the Prisma schema though, this is a temporary limitation which you can [track on GitHub](https://github.com/prisma/prisma/issues/1111).`

`Default values are not allowed on relation fields in the Prisma schema. Note however that you can still define default values on the fields backing a relation (the ones listed in the fields argument in the @relation attribute).`

Reply:
Is this your suggestion @do4gr ?

```suggestion
- Default values are not allowed on relation fields in the Prisma schema. Note however that you can still define default values on the fields backing a relation (the ones listed in the fields argument in the @relation attribute).
```

Reply:
Yes, that is correct. Maybe even add `A default value on the field backing a relation will mean that relation is populated automatically for you.`